### ISSUE TITLE
DOC501 / DOC503 not reported when return type is missing

### ISSUE THREAD

**Issue by Apakottur** [jsh9/pydoclint#170]
Consider the following code:
```python
# test.py
def func() -> int:
    """
    Docstring.

    Returns:
        int
    """
    raise ValueError
```

Running `pydoclint --style google test.py` gives a bunch of errors (correctly):
```text
    2: DOC203: Function `func` return type(s) in docstring not consistent with the return annotation. Return annotation types: ['int']; docstring return section types: ['']
    2: DOC501: Function `func` has "raise" statements, but the docstring does not have a "Raises" section
    2: DOC503: Function `func` exceptions in the "Raises" section in the docstring do not match those in the function body Raises values in the docstring: []. Raised exceptions in the body: ['ValueError'].
```

However if we remove the "Returns" section:
```python
# test.py
def func() -> int:
    """
    Docstring.
    """
    raise ValueError
```

Running `pydoclint --style google test.py` gives no errors at all, even though DOC501/DOC503 are still violated.

Is this by design? How can I check for DOC501/DOC503 even if there is no "Returns" section (which happens for example if the return type is None).

**Comment by jsh9** [comment]
Hi @Apakottur , yes, this is by design.

By default, an option called `--skip-checking-short-docstrings` is True (https://jsh9.github.io/pydoclint/config_options.html#6---skip-checking-short-docstrings-shortform--scsd-default-true).

In your example, if you remove the "returns" section, it becomes a short docstring, and no checking is run on that function.

**Comment by Apakottur** [comment]
Makes a lot of sense, thanks!
I've set the flag to false and I get the behavior that I wanted.

One exception is abstract methods of the following type:
```python
@abstractmethod
def func(self) -> None:
    """Some explanation"""
    raise NotImplementedError
```

Is there some flake8/doclint configuration magic I can do to exclude these methods from these error codes?

**Comment by Apakottur** [comment]
I've found a solution - replacing `raise NotImplementedError` with `...`.

Closing the issue.
